lib/gstreamer/va/util.py

```diff
@@ -72,6 +72,13 @@ def mapprofile(codec, profile):
       "multiview-high"        : "multiview-high",
       "stereo-high"           : "stereo-high",
     },
+    "hevc-8"   : {
+      "main"                  : "main",
+      "main-10"               : "main-10",
+      "main-12"               : "main-12",
+      "main-422-10"           : "main-422-10",
+      "main-422-12"           : "main-422-12",
+    },
```

Comment:
10 and 12 bit profiles do not go in "hevc-8".
Also, this work is already being done in #412 and will conflict with it.